cometd-java/cometd-java-benchmark/cometd-java-benchmark-client/src/main/java/org/cometd/benchmark/client/CometDLoadClient.java

```diff
@@ -80,7 +83,14 @@
 public class CometDLoadClient implements MeasureConverter {
     private static final String START_FIELD = "start";
 
-    private final AtomicHistogram histogram = new AtomicHistogram(TimeUnit.MICROSECONDS.toNanos(1), TimeUnit.MINUTES.toNanos(1), 3);
+    private final Collection<Histogram> allHistograms = new CopyOnWriteArrayList<>();
+    private final ThreadLocal<Histogram> histogramTl = ThreadLocal.withInitial(() ->
```

Comment:
Rename field to just `histogram` and put brace at EOL.

Reply:
Ok.